frontend/yarn.lock

```diff
@@ -3643,91 +3664,191 @@ es-to-primitive@^1.2.1:
     is-date-object "^1.0.1"
     is-symbol "^1.0.2"
 
+esbuild-android-64@0.14.54:
```

Comment:
I ran `yarn install -D @types/vite-plugin-react-svg`. I dunno why yarn suddenly wants all the esbuild binaries. (We could add an `overrides` and redirect these to dummy packages)